src/Service/RDSDataService/composer.json

```diff
@@ -1,13 +1,14 @@
 {
-    "name": "async-aws/lambda",
+    "name": "async-aws/rds-data-service",
```

Comment:
It seams like you removed a bunch of files from Lambda.

Reply:
I noticed 😅
I copied some boilerplate form the lambda branch and at some time got confused in the branches. I'll modify the commit when i make this pull request final.